src/zscript/damnums/damnum.txt

```diff
@@ -59,7 +60,9 @@ class DamNum : Actor
 	/*
 	 * Spawn damage numbers. Does what it says on the tin.
 	 */
-	static int spawnDamageNumbers(Actor source = null, Actor target = null, int damage = 0, Name damagetype = 'Default') {
+	static int spawnDamageNumbers(Actor source = null, Actor target = null, int damage = 0, Name damagetype = 'Default', 
+		bool usePos = false, Vector3 dnpos = (0,0,0)) 
+	{
```

Comment:
I'll probably simplify this post-merge -- I'd love to get rid of `source` now that we have `position`, but I then may need to use "consoleplayer" to offset the position a bit. I'll monkey around with it laterish.

Just commenting. No action required here. :P